rollup/circuitcapacitychecker/impl.go

```diff
@@ -74,7 +74,7 @@ func (ccc *CircuitCapacityChecker) ApplyTransaction(traces *types.BlockTrace) (*
 		C.free(unsafe.Pointer(tracesStr))
 	}()
 
-	log.Debug("start to check circuit capacity for tx", "id", ccc.ID, "TxHash", traces.Transactions[0].TxHash)
+	log.Debug("start to check circuit capacity for tx", "id", ccc.ID, "TxHash", traces.Transactions[0].TxHash, "trace", string(tracesByt))
```

Comment:
These traces are very large. Maybe `log.Trace` is more suitable?

Reply:
`log.Trace` comes with very tedious logs like very detail p2p logs.
and will make a whole log larger 

we also rotate the logs periodically,  if the logs is too long we will lose a lot of it.

and we only enable debug level for a very short term.

To me `log.Debug` seems more suitable

Reply:
Yeah, but we might want to enable debug logs to debug other issues, which then would become very hard since the logs will be filled with these traces. How about we add `miner_setCCCDebugMode(bool)`? (see `setEtherbase` function inside `worker`)

Reply:
sounds good